backend/factory/config.go

```diff
@@ -31,14 +31,14 @@ type Info struct {
 }
 
 type Configuration struct {
-	Mongodb              *Mongodb    `yaml:"mongodb"`
-	RocEnd               *RocEndpt   `yaml:"managedByConfigPod,omitempty"` // fetch config during bootup
-	LteEnd               []*LteEndpt `yaml:"endpoints,omitempty"`          // LTE endpoints are configured and not auto-detected
-	TLS                  *TLS        `yaml:"tls"`
```

Comment:
I think you unintentionally deleted this line

Reply:
Indeed. Good catch. Fixed.